### DISCUSSION TITLE
Lists Should Have Collective Actions

### DISCUSSION THREAD

**Discussion by H-Tevin** [community/community#53240] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Lists are useful in their current state, but it would be nice to have the ability to set group settings for all repos in a list. For example, I may have one list of repos that I want notifications for and another one that I don't. Letting users set list-level options (which can be overridden in each repo) would be super helpful.

**Comment by 09390812924** [comment]
![99325533-69b5-4516-a06a-d0aa13802488.JPG](https://github.com/community/community/assets/59332688/8640454a-cdb8-46a2-aa88-27551600ddb5)

**Comment by jarfam** [comment]
Wav

**Reply by Krsudwa** [reply]
I

**Reply by Krsudwa** [reply]
21gramowinsmk@gmail.com

**Reply by Krsudwa** [reply]
> 21gramowinsmk@gmail.com

@H-Tevin

**Reply by Krsudwa** [reply]
> Waverley

**Reply by jarfam** [reply]
> Wav

I

**Reply by Krsudwa** [reply]
I have attached my 87-700 - @Krsudwa []()![20230916_214351.jpg](https://github.com/community/community/assets/144963696/a4fbaea5-b39d-438c-b8f1-d8ab47773152)

21gramowinsmk@gmail.com []()I ```
- [x] Witam __ @09390812924 I think that it's good profile on that kind of - [ ] story []()![regulamin-swiadczenia-uslug-maklerskich-marzec-2023-pol.pdf](https://github.com/community/community/files/12642206/regulamin-swiadczenia-uslug-maklerskich-marzec-2023-pol.pdf)

- [ ] - # 
```

**Reply by Krsudwa** [reply]
21GRAMOWINSMK@GMAIL.COM

**Reply by jarfam** [reply]
@jarfam

**Comment by Krsudwa** [comment]
21gramowinsmk@gmail.com

**Reply by Krsudwa** [reply]
Mikado

**Reply by Krsudwa** [reply]
I

**Reply by Krsudwa** [reply]
I

**Reply by Krsudwa** [reply]
21gramowinsmk@gmail.com is different from the days of your arrival

**Reply by Krsudwa** [reply]
21gramowinsmk@gmail.com

**Comment by Krsudwa** [comment]
21gramowinsmk@gmail.com

**Reply by Krsudwa** [reply]
21gramowinsmk@gmail.com

**Comment by Krsudwa** [comment]
https://github.com/community/community/discussions/53240#discussioncomment-7024960

**Reply by Krsudwa** [reply]
https://github.com/orgs/community/discussions/53240

**Comment by Krsudwa** [comment]
21gramowinsmk@gmail.com

**Reply by Krsudwa** [reply]
21gramowinsmk@gmail.com

**Comment by MKSHAKAL** [comment]
> ### Select Topic Area
> 
> Product Feedback
> 
> ### Body
> 
> Lists are useful in their current state, but it would be nice to have the ability to set group settings for all repos in a list. For example, I may have one list of repos that I want notifications for and another one that I don't. Letting users set list-level options (which can be overridden in each repo) would be super helpful.
Gameplay

**Comment by mateuszkoltan** [comment]
Ok

**Comment by Al3676** [comment]
Oh oh thanks all support very good

**Comment by Dkkornienko** [comment]
```
``
```

**Comment by Bartletti** [comment]
⚡️⚡️⚡️⚡️
